server/app/annotations/BindingAnnotations.java

```diff
@@ -18,4 +19,12 @@ public final class BindingAnnotations {
   @Target({FIELD, PARAMETER, METHOD})
   @Retention(RUNTIME)
   public @interface Now {}
+
+  /**
+   * Holds the {@link play.i18n.Messages} object associated with the "en-US" {@link play.i18n.Lang}.
+   */
+  @Qualifier
+  @Target({FIELD, PARAMETER, METHOD})
+  @Retention(RUNTIME)
+  public @interface EnUs {}
```

Comment:
Suggestion: qualify this as a language.  EnUsLang maybe.

EnUs is on the edge of looking random if you just come across it, and if say we did portuguese instead @Pt would be incredibly unreadable on a method.

Reply:
done